### ISSUE TITLE
[FEATURE] Documentation: architecture

### ISSUE THREAD

**Issue by johnfelipe** [FlowiseAI/Flowise#3685]
Can u share me architecture flow or documentation about architecture
Tnk

**Comment by HenryHengZJ** [comment]
architecture of the whole Flowise application?

**Comment by johnfelipe** [comment]
yes pls, with drawio, mermaid, not matter format, is for comparison with other similar platforms
